src/Common/tests/System/Net/Http/GenericLoopbackServer.cs

```diff
@@ -37,9 +38,34 @@ public abstract class GenericLoopbackServer : IDisposable
         // Accept a new connection, process a single request and send the specified response, and gracefully close the connection.
         public abstract Task<HttpRequestData> HandleRequestAsync(HttpStatusCode statusCode = HttpStatusCode.OK, IList<HttpHeaderData> headers = null, string content = null);
 
+        // Accept a new connection, and hand it to provided delegate.
+        public abstract Task AcceptConnectionAsync(Func<GenericLoopbackConnection, Task> funcAsync);
```

Comment:
should it just return `Task<GenericLoopbackConnection>`?

Reply:
not this one as it does not return connection but executes delegate with the connection. See http/1 in Loopback.cs